rest/src/main/java/org/jboss/pnc/rest/provider/ProductMilestoneProvider.java

```diff
@@ -91,4 +122,118 @@ public void removeDistributedArtifact(Integer milestoneId, Integer artifactId) t
         repository.save(milestone);
     }
 
+    @Override
+    public void update(Integer id, ProductMilestoneRest restEntity) throws ValidationException {
```

Comment:
TBH I don't know.
We're currently "releasing" a milestone by setting its end date, I only added starting a release in case the date is set.